Specs/Data/Models/GltfLoader/BoxInstanced/glTF/box-instanced.gltf

```diff
@@ -16,19 +16,21 @@
           "box": {
```

Comment:
I just realized I didn't update the models in Apps/SampleData, I should check that (and double-check the sandcastles while I'm at it)